### ISSUE TITLE
Display item names when trying to buy an item

### ISSUE THREAD

**Issue by ffjdksl** [KaiKikuchi/QuickShop#110]
I think displaying the CUSTOM name of the ítem in the text you receive when you left click the shop (trying to buy) may be a good idea, this might help those custom servers with weapons and stuff, to prevent scams like players selling a normal diamond sword instead of a custom enchanted diamond sword

**Comment by KaiKikuchi** [comment]
You may want to improve your suggestion by providing screenshots and a more detailed explanation.

**Comment by smmmadden** [comment]
Hi Kai, I think what Voiddar is pointing out also covers this scenario too where we have new blocks in 1.12, but the description isn't helpful to the players:
![image](https://user-images.githubusercontent.com/10309346/27505604-22a2e0ce-5872-11e7-9a3d-55c9439dfdd0.png)
I started adding the Concrete blocks in 1.12, but instead of showing Yellow Concrete or Yellow Concrete Powder, we see Concrete:4 and Concrete:7 which to players they won't know which one is which.
Since the sign can't fit it all, it should be displayed in full on the chat window.
![image](https://user-images.githubusercontent.com/10309346/27505644-da0d772e-5872-11e7-9a44-901093afdc3d.png)

**Comment by KaiKikuchi** [comment]
Yeah in that case you'll have to edit the config file and set the names manually.

https://github.com/KaiKikuchi/QuickShop/blob/master/src/main/resources/config.yml#L101-L102

Example:
`- CONCRETE*4;DarkRedConcrete;Dark Red Concrete`
